tests/test.py

```diff
@@ -25,6 +25,7 @@ def test_ctor_dtor(self):
         self.assertDemangles('_ZN3fooD0E', 'foo::{deleting dtor}')
         self.assertDemangles('_ZN3fooD1E', 'foo::{dtor}')
         self.assertDemangles('_ZN3fooD2E', 'foo::{base dtor}')
+        self.assertDemangles('_ZN3fooC1IcEEc', 'foo::{ctor}<char>(char)')
```

Comment:
Could you add tests for destructors and casts too?

Reply:
Fixed, and good thing, because there was a bug with cast operator templates :P